### PR TITLE
More data

### PR DESCRIPTION
**Pull request by Jessica-Mach**
Added more data to make the website more robust

### CODE REVIEW

File: client/components/Navbar.js [hunk 1]
```diff
@@ -30,23 +30,45 @@ const NavBar = props => {
...
+      </li>
+    ) 
   })
-
+  
   return(
     <div>
+      <div id="top-header">
+        <div>
+          {/* <img src={require("../assets/images/logo.png")} alt="logo" /> */}
```
Comment (safang99) [inline client/components/Navbar.js]: Is this commented out code that we can remove?

File: client/components/PetShow.js [hunk 2]
```diff
@@ -26,15 +28,35 @@ const PetShow = props => {
     fetchPetDetails()
   }, []);
 
+  let formHolder;
+  if(showForm) {
+    formHolder = <AdoptionForm />;
+  }
+  const handleClick = event => {
```
Comment (safang99) [inline client/components/PetShow.js:36]: add an empty line between 34 & 35

File: client/assets/scss/main.scss [hunk 3]
```diff
@@ -12,3 +16,180 @@
...
+  h3 {
+    font-family: 'Poppins', sans-serif;
+    color: $navy-blue;
+    font-weight: bold;
+  }
+  h6 {
+    color: gray;
+  }
+}
+
```
Comment (safang99) [inline client/assets/scss/main.scss:57]: Add spaces between the nested codes blocks in this file.  For example an empty line between 45&46 and 50&51, so we end up with 

```scss
#top-header {
  display: flex;
  justify-content: space-between;
  padding: 10px 5%;
  
  h3 {
    font-family: 'Poppins', sans-serif;
    color: $navy-blue;
    font-weight: bold;
  }
  
  h6 {
    color: gray;
  }
}
```

Please apply these principles to all the code blocks below too

**Review by safang99 — CHANGES_REQUESTED** [review]
Looks great! One clarifying question and just some formatting to fix

File: client/components/Navbar.js (hunk 1 above)
Comment (Jessica-Mach) [reply client/components/Navbar.js]: Yes. Will remove. Serge may try adding the logo in later, but it will be on a separate branch.

**Pull request merged**